test/jobs/payout/enqueue_top_referrer_payout_job_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class EnqueueTopReferrerPayoutJobTest < ActiveJob::TestCase
+  test 'it enqueues only top referrers to to call EnqueuePublishersForPayoutJob' do
+    Sidekiq::Worker.clear_all
```

Comment:
Since we've done this twice recently, maybe we can move this to a global before call for all ActiveJob related tests?